test/cql-pytest/run-cassandra

```diff
@@ -48,7 +48,14 @@ def run_cassandra_cmd(pid, dir):
     os.mkdir(os.path.join(dir, 'hints'))
     env = { 'CASSANDRA_CONF': confdir,
             'CASSANDRA_LOG_DIR': logsdir,
-            'CASSANDRA_INCLUDE': '' }
+            'CASSANDRA_INCLUDE': '',
+            # Unfortunately, Cassandra's JMX cannot listen only on a specific
+            # interface. To allow tests to use JMX (nodetool), we need to
+            # have it listen on 0.0.0.0 :-( This is insecure, but arguably
+            # can be forgiven for test enviroments. The following JVM_OPTS
+            # configures that:
+            'JVM_OPTS': '-Dcassandra.jmx.remote.port=7199'
```

Comment:
Future suggestion: python allows adding a `,` to the last element as well, avoiding the need to change the last line when adding a new one.

Reply:
You're right. I can add this final comma in the new line I'm adding.